src/modules/RunUser/RunUser.ts

```diff
@@ -54,9 +54,8 @@ class RunUser extends TagoIOModule<GenericModuleParams> {
   }
 
   /**
-   * Edit Run user info
+   * Edit Run user associated data
    * @param tagoIORunURL TagoIO Run url without http
-   * @param userChangesObj Data to change in user
    */
```

Comment:
Removed the param from the documentation but the function still uses it.

`@param userChangesObj Object with the changes to be applied to the user's settings`